docs/index.md

```diff
@@ -43,13 +43,13 @@ Tide services are responsible for the following:
 * The [Lighthouse Server](services/lighthouse-server.md) reads messages from a queue and runs Google Lighthouse reports against the themes only, then sends the results back to the Tide API.
 * The [wptide.org](https://wptide.org) website includes both the documentation you are reading and also serves the JSON API endpoints from where the reports can be queried. Visit https://wptide.org/search to see examples of JSON reports.
 
-In addition we are working on the meta repository of WordPress.org to handle fetching the reports generated by Tide and showing summaries of them in the WordPress.org plugin and theme directories.
+Besides, we are working on the meta-repository of WordPress.org to handle fetching the reports generated by Tide and showing summaries of them in the WordPress.org plugin and theme directories.
```

Comment:
This change is confusing and doesn't seem to make the text more legible, please revert this change.